### ISSUE TITLE
How do you reset the position of items in list after splicing it?

### ISSUE THREAD

**Issue by StanislavAlpatiev** [gordonmzhu/questions#382]
https://watchandcode.com/courses/60264/lectures/900207

So my question is about the position of item in our list, in your case its called "todos".

we started of with the list: todos["item1", "item2", "item3", "item4", "item5",] 

when you splice it like this:  todos.splice(0, 1)
you list looks like this:  todos["item2", "item3", "item4", "item5",] 

However, "item2" still has the position of todos[1], eventhough it is technically at position todos[0] now. 

How can you reset the position of the items in the list so that the new list after having been altered with the splice method can take on positions of the newly outputed list?

Kind regards Stanislav

**Comment by gordonmzhu** [comment]
What do you mean ""item2" still has the position of todos[1]"? I don't understand what that means. Can you rephrase?

**Comment by StanislavAlpatiev** [comment]
if you want to reference "item2" after splicing the original list that had 5 items, you would have to call it with todos[1]. My issue is that it hold the position of todos[0] if you would enter the list that remains after the splice into memory again.  So how do i make it so that "item2" is referenced by todos[0] and not by todos[1] anymore. Same goes for all the other items, meaning that all have their [x] become [x-1] in accordance with the new list.

Hope this is clearer. Sorry if not 
Kind regards Stanislav

**Comment by StanislavAlpatiev** [comment]
Sorry i found out that the
